config/jobs/etcd-druid/etcd-druid-api-unit-tests.yaml

```diff
@@ -0,0 +1,64 @@
+presubmits:
+  gardener/etcd-druid:
+    - name: pull-etcd-druid-api-unit
+      cluster: gardener-prow-build
+      always_run: true
+      optional: true
+      skip_branches:
+        - hotfix-v\d+.\d+ # don't run on release branches for now (add a job per branch later)
+      decorate: true
+      decoration_config:
+        timeout: 40m
+        grace_period: 10m
+      annotations:
+        description: Runs unit tests for etcd-druid API in pull requests
+        fork-per-release: "true"
+      spec:
+        containers:
+          # Run all tests sequentially in one container or as separate prow jobs.
+          # Test will fail when 'check-generate' (which includes a revendor) and 'test' or 'check' run in parallel on the same volume
+          - image: europe-docker.pkg.dev/gardener-project/releases/ci-infra/golang-test:v20250129-fcd6b72-1.23
+            command:
+              - bash
+            args:
+              - c
+              - "cd api && make check check-generate test-unit"
+            resources:
+              limits:
+                memory: 16Gi
+              requests:
+                cpu: 4
+                memory: 8Gi
+periodics:
+  - name: ci-etcd-druid-api-unit
+    cluster: gardener-prow-build
+    interval: 4h
+    extra_refs:
+      - org: gardener
+        repo: etcd-druid
+        base_ref: master
+    decorate: true
+    decoration_config:
+      timeout: 40m
+      grace_period: 10m
+    annotations:
+      description: Runs unit tests for etcd-druid periodically
+      testgrid-dashboards: gardener-etcd-druid
+      testgrid-days-of-results: "60"
+      #fork-per-release: "true"
+    spec:
+      containers:
+        # Run all tests sequentially in one container or as separate prow jobs.
+        # Test will fail when 'check-generate' (which includes a revendor) and 'test' or 'check' run in parallel on the same volume
+        - image: europe-docker.pkg.dev/gardener-project/releases/ci-infra/golang-test:v20250129-fcd6b72-1.23
+          command:
+            - bash
+          args:
+            - c
+            - "cd api && make check check-generate test-unit"
+          resources:
+            limits:
+              memory: 16Gi
+            requests:
+              cpu: 4
+              memory: 8Gi
```

Comment:
As long as we are in evaluation mode and your etcd-druid PR is not merged yet, we don't really need to test it on master branch do we? It will fail all the time anyway.

Reply:
ok i will remove the `periodics`